MODULE.bazel

```diff
@@ -15,13 +15,36 @@ module(
 bazel_dep(name = "abseil-cpp", version = "20230802.0.bcr.1", repo_name = "com_google_absl")
 bazel_dep(name = "bazel_skylib", version = "1.4.1")
 bazel_dep(name = "jsoncpp", version = "1.9.5")
+bazel_dep(name = "platforms", version = "0.0.8")
 bazel_dep(name = "rules_cc", version = "0.0.9")
-bazel_dep(name = "rules_java", version = "5.3.5")
-bazel_dep(name = "rules_jvm_external", version = "5.1")
+bazel_dep(name = "rules_java", version = "6.0.0")
+bazel_dep(name = "rules_jvm_external", version = "6.0")
 bazel_dep(name = "rules_pkg", version = "0.7.0")
 bazel_dep(name = "rules_python", version = "0.10.2")
-bazel_dep(name = "platforms", version = "0.0.8")
 bazel_dep(name = "zlib", version = "1.2.11")
 
 # TODO: remove after toolchain types are moved to protobuf
-bazel_dep(name = "rules_proto", version = "4.0.0")
\ No newline at end of file
+bazel_dep(name = "rules_proto", version = "4.0.0")
+
+maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+maven.install(
+    artifacts = [
```

Comment:
Can you explain why these artifacts are needed?

Reply:
This mirrors the WORKSPACE https://github.com/protocolbuffers/protobuf/blob/a978b75794a6ce4547c9db08a115c458d9190934/WORKSPACE#L61-L70, seems like the java targets just need these as typical third party deps? https://github.com/protocolbuffers/protobuf/blob/a978b75794a6ce4547c9db08a115c458d9190934/java/util/BUILD.bazel#L16-L20

https://github.com/protocolbuffers/protobuf/blob/a978b75794a6ce4547c9db08a115c458d9190934/java/util/src/main/java/com/google/protobuf/util/JsonFormat.java#L14

Reply:
thanks!